tests/test_adjoint_jacobian.py

```diff
@@ -694,6 +694,64 @@ def circuit(x, y, z):
 
         assert np.allclose(grad_A, grad_F, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize(
+        "operation",
+        [
+            qml.PhaseShift,
+            qml.RX,
+            qml.RY,
+            qml.RZ,
+            qml.IsingXX,
+            qml.IsingXY,
+            qml.IsingYY,
+            qml.IsingZZ,
+            qml.SingleExcitation,
+            qml.SingleExcitationMinus,
+            qml.SingleExcitationPlus,
+            qml.DoubleExcitation,
+            qml.DoubleExcitationMinus,
+            qml.DoubleExcitationPlus,
+            qml.GlobalPhase,
```

Comment:
is that the list of all supported operations? :thinking:

Reply:
Good spot, some implemented generators are missing here. Let me add them.